shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/scenario/general/mysql.xml

```diff
@@ -62,7 +64,7 @@
     </create-table-order-item>
     
     <full-insert-order>
-        INSERT INTO
+        INSERT IGNORE INTO
```

Comment:
It's better not add `IGNORE`, else we could not know when ISNERT SQL execution failed